src/reducers/appInstance.js

```diff
@@ -11,6 +11,7 @@ import { showErrorToast } from '../utils/toasts';
 
 const DEFAULT_SETTINGS = {
   headerVisible: false,
+  spaces: null,
```

Comment:
This should probably be `[]` by default.

Reply:
I purposely used `null` to detect the initial state, and set up at the application first loading the `spaceId`. This allows to not include the current space (if it is something we want to achieve). Otherwise I can force `spaceId` to always be included and the user cannot unselect it.